kore/src/Kore/Repl/State.hs

```diff
@@ -416,7 +416,10 @@ liftSimplifierWithLogger mLogger simplifier = do
     (textLogger, maybeHandle) <- logTypeToLogger logType
     let logger =
             Log.koreLogFilters koreLogOptions
-            $ Log.makeKoreLogger timestampsSwitch textLogger
+            $ Log.makeKoreLogger
+                (Log.ExeName "kore-repl")
```

Comment:
I think you can get the executable name from the `koreLogOptions` in the state instead of adding the string directly here.